JoinColony/colonyCDapp
# Implement unlock token multi-sig motion

## Description

- Allow users to be able to create, finalize and reject a multi-sig unlock token motion.

Block ingestor changes here: https://github.com/JoinColony/block-ingestor/pull/244

## Testing

* Restart your dev stack after checking out the branch, and then ensure that the multi sig extension is enabled and the threshold is set to 2 in the extension settings.
* Create a new action of type Unlock Token and choose multisig as the permissions method. It should create successfully, and you should be able to reject the motion straight away as the owner.
* If you rejected it, create a new one. Give another user owner permissions with multisig authority. Log in to that other user in another window, and approve the motion you created.
* You should now be able to finalize the motion.

## Diffs

**New stuff** ✨

* New block ingestor hash
* New prop for DecisionMethod row of completed actions `isMultiSig`

**Changes** 🏗

* Turn on the completed action sidebar panel UI for unlock token, and add the multisig meatball menu there so that you can reject.

Resolves #2187
